units/user/systemd-exit.service

```diff
@@ -14,3 +14,6 @@ DefaultDependencies=no
 Requires=shutdown.target
 After=shutdown.target
 SuccessAction=exit-force
+
+[Service]
+Slice=-.slice
```

Comment:
Hmm, shutdown should be blocked by an .slice unit. Weed to fix that in the manager anyway.

Also:
systemd-tmpfiles-clean.service
systemd-tmpfiles-clean.timer
systemd-tmpfiles-setup.service
→ those should be in the background.slice.

Reply:
Only `systemd-tmpfiles-clean.service`, I would say. The timer doesn't have it and `setup` runs early and needs to get its job done. Added a commit to set it.